Content.Server/Speech/EntitySystems/SkeletonAccentSystem.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Text.RegularExpressions;
+using Content.Server.Speech.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class SkeletonAccentSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    private static readonly Dictionary<string, string> DirectReplacements = new()
+    {
+        { "fuck you", "I've got a BONE to pick with you" },
+        { "fuck", "rattle rattle" },
+        { "shit", "rattle rattle" },
+        { "definitely", "make no bones about it" },
+        { "absolutely", "make no bones about it" },
+        { "lonely", "bonely"}
```

Comment:
perhaps consult discord/ingame for more ideas for replacements, i got a lot of good ones out of doing that for the mobster accent

Reply:
Great idea thanks! Have asked in #ideaguys, and will update with any good ones I see. If any good ones come in after Marge, I can always re-PR as a batch.